fastlane/Fastfile

```diff
@@ -47,15 +47,46 @@ end
   lane :code_freeze do | options |
     old_version = ios_codefreeze_prechecks(options)
     
-    ios_bump_version_release(skip_glotpress: true)
+    ios_bump_version_release()
     new_version = ios_get_app_version()
+    ios_update_release_notes(new_version: new_version)
     setbranchprotection(repository:GHHELPER_REPO, branch: "release/#{new_version}")
     setfrozentag(repository:GHHELPER_REPO, milestone: new_version)
 
+    ios_localize_project()
     ios_tag_build()
     get_prs_list(repository:GHHELPER_REPO, start_tag:"#{old_version}", report_path:"#{File.expand_path('~')}/simplenotemacos_prs_list_#{old_version}_#{new_version}.txt")
   end
 
+  #####################################################################################
+  # update_appstore_strings
+  # -----------------------------------------------------------------------------------
+  # This lane updates the AppStoreStrings.pot files with the latest content from
+  # the release_notes.txt file and the other text sources
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane update_appstore_strings version:<release note version>
+  #
+  # Example:
+  # bundle exec fastlane update_appstore_strings version:1.1
+  #####################################################################################
+  desc "Updates the AppStoreStrings.pot file with the latest data"
+  lane :update_appstore_strings do | options |
+    prj_folder = Pathname.new(File.join(Dir.pwd, "..")).expand_path.to_s
+    source_metadata_folder = File.join(prj_folder, "fastlane/appstoreres/metadata/source")
+
+    files = {
+      whats_new: File.join(prj_folder,  "/Simplenote/Resources/release_notes.txt"),
+      app_store_subtitle: File.join(source_metadata_folder, "subtitle.txt"),
+      app_store_desc: File.join(source_metadata_folder, "description.txt"),
+      app_store_keywords: File.join(source_metadata_folder, "keywords.txt")
+    }
+
+    ios_update_metadata_source(po_file_path: prj_folder + "/Simplenote/Resources/AppStoreStrings.pot",
```

Comment:
Nitpick:

```suggestion
    ios_update_metadata_source(po_file_path: File.join(prj_folder, "/Simplenote/Resources/AppStoreStrings.pot"),
```

- The rest of the lane uses `File.join`
- `File.join` is smart enough to avoid joining paths resulting in `//`, i.e. `/a/b/` and `/c/d` would be `/a/b/c/d`, but if we just use `+` it'd be an invalid path, `/a/b//c/d`

<img alt="image" src="https://user-images.githubusercontent.com/1218433/88767478-091ad900-d1bd-11ea-869f-620530b33c1b.png" width=300/>

Reply:
Much better, thank you!!